spec/requests/users_controller_spec.rb

```diff
@@ -1,36 +1,36 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET #index' do
-    it 'success for users page' do
-      get users_path
-      expect(response).to have_http_status(200)
-    end
+# RSpec.describe 'Users', type: :request do
+#   describe 'GET #index' do
+#     it 'success for users page' do
+#       get users_path
+#       expect(response).to have_http_status(200)
+#     end
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented lines of code as it makes your code look untidy 👍